### PR TITLE
GL Viewer UI + GLFW

### PR DESCRIPTION
**Pull request by qhdwight**
- IMGUI for quick and easy photoshop-like interface
- GLFW instead of GLUT (newer, well maintained, more features but not too bloated)
- Refactor camera class to be much simpler, also added basic first-person camera movement with arrow keys
- Fix misc. bugs

### CODE REVIEW

**Comment by niwhsa9** [comment]
theres still something wacky hapenning during playback, the files seem to be coming up out of order

File: jetson/percep_obs_detect/src/viewer.cpp [hunk 1]
```diff
@@ -352,11 +352,11 @@ void Viewer::keyCallback(GLFWwindow* window, int key, int scancode, int action,
     if (action == GLFW_PRESS) {
         switch (key) {
             case GLFW_KEY_A: {
-                viewer->frame++;
+                viewer->frame--;
```
Comment (qhdwight) [inline jetson/percep_obs_detect/src/viewer.cpp:355]: Wait why does A key go backwards in time

**Comment by qhdwight** [comment]
image

**Review by niwhsa9 — APPROVED** [review]
really excellent work, squash and merge when ready.

**Comment by niwhsa9** [comment]
note: this code pushes the percepDev container to version 1.5.1

**Pull request merged**